specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -304,43 +321,78 @@ _jakarta.xml.rpc.handler.MessageContext_ type.]
 
 
 [[a2331]]
-==== Pruned Java Technologies
+==== Optional Jakarta Technologies
 
 As the Jakarta EE specification has evolved,
 some of the technologies originally included in Jakarta EE are no longer as
 relevant as they were when they were introduced to the platform. The
-Jakarta EE expert group follows a process first defined by the Java SE
-expert group ( _http://mreinhold.org/blog/removing-features_ ) to prune
+Jakarta EE Platform Specification Project follows a process similar to the one first defined by the Java SE
+expert group ( _http://mreinhold.org/blog/removing-features_ ) to stabilize and deprecate
 technologies from the platform in a careful and orderly way that
 minimizes the impact to developers using these technologies, while
-allowing the platform to grow even stronger. In short, the process
-defines two steps:
+allowing the platform to grow even stronger. In short, our process
+defines three steps:
+
+. The Platform Specification Project for release N 
+of the platform decides to propose that a particular feature be
+marked "proposed optional". The specification for that release documents the proposal.
+. The Platform Specification Project for release N+1 decides whether to
+mark the feature as "optional" for this N+1 release, retain it as a required component,
+or leave it in the "proposed optional" state for the next N+2 release cycle to decide.
+. Eventually, the Platform Specification Project for release N+2 (or beyond) can decide
+to officially "prune" (remove) the "optional" feature from the Platform.
 
+The result of successfully applying this
+policy to a feature is to allow a gradual deprecation and possible removal
+of the feature from a required component of the platform.
+Product vendors can _choose_ to remove an "optional" feature from their products.
+Product vendors are _required_ to remove a "pruned" feature from their products for
```

Comment:
Drop statement that "Product vendors are _required_ to remove a "pruned" feature from their products for"

Reply:
+1, thanks!

Reply:
Reworded instead of complete removal of this statement. I want to be clear that "removed" features are no longer maintained. And, that any integration of a "removed" feature with the Platform is left as an exercise for the vendor. That is, it is not defined by the Spec.